secondLayout.js

```diff
@@ -0,0 +1,317 @@
+// Getting the canvas from html page
```

Comment:
I think this should be called main.js or something since this is starting the whole game including the first page and everything